libraries/botbuilder-dialogs-adaptive/src/input/inputDialog.ts

```diff
@@ -359,7 +359,9 @@ export abstract class InputDialog extends Dialog implements InputDialogConfigura
             msg = await this.prompt.bind(dc, dc.state);
         }
 
-        msg.inputHint = InputHints.ExpectingInput;
+        if (typeof msg?.inputHint !== 'string' || !msg.inputHint) {
```

Comment:
I think `if (typeof msg?.inputHint !== 'string')` should cover all the cases, unless `msg.inputHint = ''` should be overridden to `InputHints.ExpectingInput`.

Reply:
yes. Empty inputHint should be overridden to the default ExpectingInput. Thanks.